### PR TITLE
Update categoriesHighlights README

### PR DESCRIPTION
**Pull request by paul0vinicius**
#### What is the purpose of this pull request?

As the title says.

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/CategoriesHighlights/README.md [hunk 1]
```diff
@@ -1,31 +1,64 @@
...
 
-## Usage
 You can use it in your code like a React component with the jsx tag: `<CategoriesHighlights />`. 
 ```jsx
 <CategoriesHighlights
     categoriesHighlighted={ 
         categoryX: { name: 'X', image: 'Image X' }, 
         categoryY: { name: 'Y', image: 'Image Y' } 
-    } 
+    } x
```
Comment (theomoura) [inline react/components/CategoriesHighlights/README.md]: x?

File: react/components/CategoriesHighlights/README.md [hunk 2]
```diff
@@ -1,31 +1,64 @@
-# CategoriesHighlights description
-CategoriesHighlights is a canonical component that any VTEX app can import.
+# Categories Highlights
+
+## Description
+
+Categories Highlights shows two or four categories in a lighlighted position in the store. This Component can be imported and used by any VTEX App.
```
Comment (theomoura) [inline react/components/CategoriesHighlights/README.md]: `lighlighted `?

Comment (theomoura) [reply react/components/CategoriesHighlights/README.md]: What do you think?

```suggestion
Categories Highlights shows two or four categories, in the form of an image, with a highlighted position in the store. This Component can be imported and used by any VTEX App.
```

File: react/components/CategoriesHighlights/README.md (hunk 1 above)
Comment (paul0vinicius) [reply react/components/CategoriesHighlights/README.md]: typing error. my bad.

File: react/components/CategoriesHighlights/README.md (hunk 2 above)
Comment (paul0vinicius) [reply react/components/CategoriesHighlights/README.md]: I wrote it this way: `Categories Highlights is a banner which shows two or four categories in a highlighted position in the store. This Component can be imported and used by any VTEX App.`

by saying it is a banner we imply that it has an image in it.

**Pull request merged**